README.md

```diff
@@ -1,4 +1,4 @@
-# Top.gg Node SDK
+# Top.<span>gg Node SDK
```

Comment:
I don't think this was intentional

Reply:
In some situations, (not GitHub readme) but some MD viewers will turn this into a link to top.gg which I've done the same for to avoid links.
Without:
![image](https://user-images.githubusercontent.com/7953076/109556152-699c2c80-7aa4-11eb-856f-29c58cb87325.png)
With:
![image](https://user-images.githubusercontent.com/7953076/109556181-76208500-7aa4-11eb-8ae5-e8f8a4192d34.png)

Reply:
@Xetera is there something distinctly wrong with it that I don't know about?

Reply:
I just don't want the source code looking messy that's all